Cargo.toml

```diff
@@ -33,6 +33,7 @@ bitvec = "1.0"
 byteorder = "1.4.3"
 thiserror = "1.0"
 halo2curves = { version = "0.1.0", features = ["derive_serde"] }
+ec-gpu-gen = "0.7.0"
```

Comment:
This PR looks good! 

I think using the code from here will avoid DensityTracker: https://github.com/microsoft/Nova/tree/main/src/bellpepper